test/IRGen/virtual-function-elimination-ir.swift

```diff
@@ -20,19 +20,19 @@ class MyDerivedClass: MyClass {
 // CHECK:      @"$s4main7MyClassCMn" =
 // CHECK-SAME:     align 4, !type !0, !type !1, !type !2, !vcall_visibility !3, !typed_global_not_for_cfi !4
 
-// Has vfunc slots at offsets 72, 80, 88 (on 64-bit) / 48, 52, 56 (on 32-bit).
+// Has vfunc slots at offsets 80, 88, 96 (on 64-bit) / 52, 56, 60 (on 32-bit).
```

Comment:
The updates to these comments are not from this change but rather left over from: https://github.com/apple/swift/commit/a8d4d57f11796d82939c4aee06efc7a2aaa61ae3